dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -1466,7 +1468,7 @@ def self.seed_from_csv(state_code, school_year, update, filename, dry_run = fals
         courses = get_courses(state_code, row_hash, school_year, update)
         # state_school_id is unique so there should be at most one school.
         school = School.where(state_school_id: state_school_id).first
-        if school && state_school_id
+        if school && state_school_id && courses
```

Comment:
Did you have to change this because of an old file being ingested? I thought this wouldn't be an issue since all new files are in V2 format?

Reply:
Correct, new files are using V2 format. 

However, if they contains pre-2019 school-year data, we have to explicitly add them to special lists such as `STATES_USING_FORMAT_V2_IN_2018_19` and `STATES_USING_FORMAT_V2_IN_2017_18`. Otherwise, seeding code will fail to extract courses info (i.e. `courses` is `nil`) but will still report success. This is a bit of unwritten knowledge which I actually forgot when importing NY and PA 2018-2019 data! By checking `courses` is `nil` or not I can detect the issue early in dry-run.